inc/Engine/Optimization/RUCSS/Controller/UsedCSS.php

```diff
@@ -700,7 +700,7 @@ protected function remove_google_font_preconnect( string $html ): string {
 		$clean_html = $this->hide_noscripts( $clean_html );
 		$clean_html = $this->hide_scripts( $clean_html );
 		$links      = $this->find(
-			'<link\s+([^>]+[\s"\'])?rel\s*=\s*[\'"]((preconnect)|(dns-prefetch))[\'"]([^>]+)?\/?>',
+			'<link\s+([^>]+[\s"\'])?rel\s*=\s*[\'"]((preconnect)|(dns-prefetch)|(stylesheet)|(preload))[\'"]([^>]+)?\/?>',
```

Comment:
shouldn't stylesheet and preload links be removed in `remove_used_css_from_html()`?